src/containers/org/people/components/RoleChipsList.js

```diff
@@ -55,8 +55,7 @@ const RoleChipsList = ({
   };
 
   const { id } = getUserProfile(member);
-  const getHandleDelete = (role :Role) => (e :SyntheticEvent<HTMLElement>) => {
-    e.preventDefault();
+  const getHandleDelete = (role :Role) => {
```

Comment:
update this function name to just handleDelete since it is no longer used to create a handler function.